go-gen-checksum.sh

```diff
@@ -14,25 +14,24 @@ GOMODULE_PATHS=("core/network/fabric-interop-cc/libs/utils"
 "core/network/fabric-interop-cc/contracts/interop")
 
 for GOMODULE in ${GOMODULE_PATHS[@]}; do
-  echo "!!!!! START !!!!!!!"
+  echo "############# START $GOMODULE ################"
   echo $GOMODULE
   pushd $ROOT_DIR/$GOMODULE
-  GOMOD_DEPS=$(go mod graph | grep "$REPO/$GOMODULE $REPO" | cut -d ' ' -f 2)
-  cp go.sum go.sum.bkp
+  GOMOD_DEPS=$((go mod graph | grep "$REPO/$GOMODULE $REPO" | cut -d ' ' -f 2) || (echo "ERROR: In generating dependency graph" && exit 1))
   popd
 
   for GOMOD_DEP in ${GOMOD_DEPS[@]}; do
-    echo "######## START MOD DEP ############"
+    echo "--------- START DEP -----------"
     GOMOD_PATH=$(echo $GOMOD_DEP | cut -d '@' -f 1 | awk -F "$REPO/" '{print $2}')
-    echo $GOMOD_DEP
-    echo $GOMOD_PATH
+    echo DEP: $GOMOD_DEP
```

Comment:
I think it's be better to put this as `echo "DEP: "$GOMOD_DEP`.

Similar changes in other lines.

Reply:
Done.